Realistic_Electric_Trains_0.1.1/control.lua

```diff
@@ -6,14 +6,14 @@ require("logic.overhead_line")
 
 -- initialization
 script.on_init(
-	function(e)
-		-- init lookup tables
-		global.wire_for_pole = {}   -- Pole ID -> Wire Entity
-		global.power_for_pole = {}  -- Pole ID -> Power Entity
-		global.graphic_for_pole = {}-- Pole ID -> Graphic Entity
-		global.power_for_rail = {}  -- Rail ID -> Power Entity
-		global.electric_locos = {}  -- Loco ID -> Loco Entity
-	end
+		function(e)
+			-- init lookup tables
+			global.wire_for_pole = {}   -- Pole ID -> Wire Entity
+			global.power_for_pole = {}  -- Pole ID -> Power Entity
+			global.graphic_for_pole = {}-- Pole ID -> Graphic Entity
+			global.power_for_rail = {}  -- Rail ID -> Power Entity
+			global.electric_locos = {}  -- Loco ID -> Loco Entity
+		end
```

Comment:
Sorry about that.. the IDE's auto indentation algorithm thought this is more correct. If you don't like it tell me and I revert that.